src/app/offers/offer-details/offer-details.component.html

```diff
@@ -182,6 +182,19 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
                     </div>
                 </div>
             </div>
+
+            <div class="left__news">
+                <ng-container *ngIf="states.news != null">
```

Comment:
if there are no news, markup will look like this:

```
<div class="left__news">
</div>
```

We don't want that. Remove ng-container and put `*ngIf="states.news != null"` on `div.left__news`.